app/assets/javascripts/event_emails.js.coffee

```diff
@@ -27,3 +27,17 @@ $("input[type='checkbox']").each ->
 
 $('#state-menu').click (event) ->
   event.stopPropagation()
+
+# To limit the ajax call to only when the content of body is changed
+flag = true
```

Comment:
Drop it, I don't think this will be an issue and makes the code more complicated